### PR TITLE
Dashboard screens: tire / brake heat view + IMU view

### LINKED ISSUE #1300: tire / brake temperature dashboard page
A dashboard page that visualizes heat gradients for 4 tires, plus optional brake rotor temperature. 

Would be organized as such.  

O = rotor temperature
[====] = tire temperature

The number of zones per tire and rotor would be configurable. 
Digital indication of each zone would also be displayed for each tire / rotor. 

```
[====]  O         O [====]


[====] O           O [====]
```

### PR DESCRIPTION
**Pull request by brentpicasso**
Resolves #1300 and completes a long advertised feature showcased on product page :-/ 
https://www.autosportlabs.com/racecapture_mk3_apex/

You can use this script to re-create the effect seen in the gif below:
https://gist.github.com/brentpicasso/2f20a1288c58b54da2719de425446b63

https://i.imgur.com/MPekupy.gif

### CODE REVIEW

File: autosportlabs/racecapture/views/dashboard/widgets/heatmapgauge.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See the GNU General Public License for more details. You should
+# have received a copy of the GNU General Public License along with
+# this code. If not, see <http://www.gnu.org/licenses/>.
+import os
+import kivy
+kivy.require('1.9.1')
```
Comment (ddimensia) [inline autosportlabs/racecapture/views/dashboard/widgets/heatmapgauge.py]: I believe ;this should be 1.10.x now

File: autosportlabs/racecapture/widgets/heat/heatgauge.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See the GNU General Public License for more details. You should
+# have received a copy of the GNU General Public License along with
+# this code. If not, see <http://www.gnu.org/licenses/>.
+
+import kivy
+kivy.require('1.9.1')
```
Comment (ddimensia) [inline autosportlabs/racecapture/widgets/heat/heatgauge.py]: Kivy 1.10.0

File: autosportlabs/racecapture/widgets/heat/wheelcorner.py [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See the GNU General Public License for more details. You should
+# have received a copy of the GNU General Public License along with
+# this code. If not, see <http://www.gnu.org/licenses/>.
+import os
+import kivy
+kivy.require('1.9.1')
```
Comment (ddimensia) [inline autosportlabs/racecapture/widgets/heat/wheelcorner.py]: Kivy 1.10.0

File: autosportlabs/racecapture/widgets/heat/wheelcorner.py [hunk 4]
```diff
@@ -0,0 +1,308 @@
+#
+# Race Capture App
+#
+# Copyright (C) 2014-2016 Autosport Labs
```
Comment (ddimensia) [inline autosportlabs/racecapture/widgets/heat/wheelcorner.py]: date

File: autosportlabs/racecapture/widgets/heat/heatgauge.py [hunk 5]
```diff
@@ -0,0 +1,190 @@
+#
+# Race Capture App
+#
+# Copyright (C) 2014-2016 Autosport Labs
```
Comment (ddimensia) [inline autosportlabs/racecapture/widgets/heat/heatgauge.py]: date

File: autosportlabs/racecapture/views/dashboard/widgets/trackmapgauge.py [hunk 6]
```diff
@@ -0,0 +1,90 @@
+#
+# Race Capture App
+#
+# Copyright (C) 2014-2016 Autosport Labs
```
Comment (ddimensia) [inline autosportlabs/racecapture/views/dashboard/widgets/trackmapgauge.py]: date

File: autosportlabs/racecapture/views/dashboard/widgets/heatmapgauge.py [hunk 7]
```diff
@@ -0,0 +1,104 @@
+#
+# Race Capture App
+#
+# Copyright (C) 2014-2016 Autosport Labs
```
Comment (ddimensia) [inline autosportlabs/racecapture/views/dashboard/widgets/heatmapgauge.py]: date

File: autosportlabs/racecapture/views/dashboard/heatmapview.py [hunk 8]
```diff
@@ -0,0 +1,187 @@
+#
+# Race Capture App
+#
+# Copyright (C) 2014-2016 Autosport Labs
```
Comment (ddimensia) [inline autosportlabs/racecapture/views/dashboard/heatmapview.py]: date

File: autosportlabs/racecapture/views/dashboard/heatmapview.py [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See the GNU General Public License for more details. You should
+# have received a copy of the GNU General Public License along with
+# this code. If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import kivy
+kivy.require('1.9.1')
```
Comment (ddimensia) [inline autosportlabs/racecapture/views/dashboard/heatmapview.py]: kivy 1.10.0

File: autosportlabs/uix/imu/imuview.py [hunk 10]
```diff
@@ -20,173 +20,283 @@
...
+        Window.unbind(on_motion=self.on_motion)
+
+    def on_model_path(self, instance, value):
+        self._setup_object()
+
+    def _setup_object(self):
+
+        self.clear_widgets()
+        shader_file = resource_find('resource/models/shaders.glsl')
+        # obj_path = resource_find('resource/models/car-formula-white.obj')
```
Comment (ddimensia) [inline autosportlabs/uix/imu/imuview.py]: Based on my reading of the code these hard coded paths should all go away.

Comment (brentpicasso) [reply autosportlabs/uix/imu/imuview.py]: fixed.

**Pull request merged**